packages/squiggle-lang/__tests__/TS/SampleSet_test.ts

```diff
@@ -0,0 +1,175 @@
+import { Distribution } from "../../src/js/index";
+import { expectErrorToBeBounded, failDefault } from "./TestHelpers";
+import * as fc from "fast-check";
+
+// Beware: float64Array makes it appear in an infinite loop.
+let arrayGen = () =>
+  fc.float32Array({
+    minLength: 10,
+    maxLength: 10000,
+    noDefaultInfinity: true,
+    noNaN: true,
+  });
+
+describe("SampleSet: cdf", () => {
+  let n = 10000;
+  test("at the highest number in the distribution is within epsilon of 1", () => {
+    fc.assert(
+      fc.property(arrayGen(), (xs) => {
+        let ys = Array.from(xs);
```

Comment:
We're not sure what this is, or why ys comes from xs. Maybe add a comment?

Reply:
Strictly a cast. `fast-check`'s `float64Array` is doesn't work with a few of the functions I want, i.e. doesn't expose `.reduce`.

Reply:
We typically use ``xs`` and ``ys`` to mean xy coordinates. Maybe just use `_xs` for one of these, then ``let xs = Array.from(_xs)``